main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+        players_info = json.load(players)
+
+    for player, info in players_info.items():
+        race, created = Race.objects.get_or_create(
+            name=info["race"]["name"],
+            description=info["race"]["description"]
+        )
+        for skill in info["race"]["skills"]:
```

Comment:
You should check if race is created and then create skills!

Reply:
thanks, will fix it